### PR TITLE
os/bluestore: using macro OBJECT_MAX_SIZE to check osd_max_object_size.

### PR DESCRIPTION
**Pull request by majianpeng**
Signed-off-by: Jianpeng Ma <jianpeng.ma@intel.com>

### CODE REVIEW

File: src/os/bluestore/BlueStore.cc [hunk 1]
```diff
@@ -4013,9 +4013,10 @@ int BlueStore::get_block_device_fsid(CephContext* cct, const string& path,
 int BlueStore::_open_path()
 {
   // sanity check(s)
-  if (cct->_conf->get_val<uint64_t>("osd_max_object_size") >=
-      4*1024*1024*1024ull) {
-    derr << __func__ << " osd_max_object_size >= 4GB; BlueStore has hard limit of 4GB." << dendl;
+  if (cct->_conf->get_val<uint64_t>("osd_max_object_size") >
```
Comment (xiexingguo) [inline src/os/bluestore/BlueStore.cc]: =？

Comment (liewegas) [reply src/os/bluestore/BlueStore.cc]: yeah, please keep it >=

**Comment by majianpeng** [comment]
sorry for late response. Update

**Comment by tchaikov** [comment]
@majianpeng need rebase

**Comment by majianpeng** [comment]
update. please review. Thanks!

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2018-01-09_16:17:21-rados-wip-sage3-testing-2018-01-09-0850-distro-basic-smithi/

**Pull request merged**